packages/panzer/adapters-ioss/test/ioss_connmngr/tIOSSConnManager.cpp

```diff
@@ -354,7 +354,7 @@ TEUCHOS_UNIT_TEST(tIOSSConnManager, basic)
 
       // Check for correct connectivity for all local elements
       if (print) output << "Checking for correct connectivity for all local elements." << std::endl;
-      int * conn;
+      panzer::Ordinal64 * conn;
```

Comment:
Should we have a `panzer::LocalOrdinal` and `panzer::GlobalOrdinal` defined, instead of `panzer::Ordinal64`? There is/was a hidden config call that changed Ordinal64 to a regular int, so maybe specializing the name would be helpful to the user. We could probably get away with deprecating the `panzer::Ordinal64` to avoid breaking Drekar/Charon/EMPIRE.

Reply:
yes - I'm going to rename Ordinal64 to GlobalOrdinal, but not until the last phase of this process. Right now we have too many classes defining template parameters named LocalOrdinal and GlobalOrdinal. Want to keep the unique name until all template instances of GlobalOrdinal are removed and then use sed to rename.